.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,316 @@
+name: Website
+
+on:
+  push:
+    branches:
+      - jekyll
+
+  pull_request:
+    branches:
+      - jekyll
+
+  workflow_dispatch:
```

Comment:
Yes, exactly. There's no required options for it. It would be beneficial if it _could_ be restricted to a set of branches. For the moment I've left a comment warning to run it on source branch only.

However, for the moment, the only options you can pass are definitions for any inputs to the workflow.